beeai/agents/triage_agent.py

```diff
@@ -53,6 +53,7 @@ class BackportData(BaseModel):
     patch_url: str = Field(description="URL or reference to the source of the fix")
     justification: str = Field(description="Clear explanation of why this patch fixes the issue")
     jira_issue: str = Field(description="Jira issue identifier")
+    cve_id: str = Field(description="CVE identifier")
```

Comment:
could you please add this also [here](https://github.com/packit/ai-workflows/blob/83884485f3f56777c96cf4dc49218bf55f6dea90/beeai/agents/triage_agent.py#L214), with a brief description? I would say the agent could be then quite smart about this, as it is already without the instruction, usually directly mentions "this is a CVE-*" in logs, I can test it more later

Reply:
I just verified the addition, and works pretty well, thanks for the suggestion!
```
INFO:__main__:Direct run completed: {
    "resolution": "backport",
    "data": {
        "package": "libxml2",
        "branch": "c9s",
        "patch_url": "https...780",
        "justification": "The Jira issue ...",
        "jira_issue": "RHEL-102805",
        "cve_id": "CVE-2025-7425"
    }
}
```